central/graphql/resolvers/common_vulnerabilities.go

```diff
@@ -13,49 +13,68 @@ import (
  * NOTE: This list is and should remain alphabetically ordered
  */
 var commonVulnerabilitySubResolvers = []string{
-	"createdAt: Time", // Discovered At System
-	"cve: String!",
+	"cveBaseInfo: CVEInfo",
 	"cvss: Float!",
 	"envImpact: Float!",
 	"fixedByVersion: String!",
 	"id: ID!",
 	"impactScore: Float!",
 	"isFixable(query: String): Boolean!",
-	"lastModified: Time",
 	"lastScanned: Time",
-	"link: String!",
-	"publishedOn: Time",
-	"scoreVersion: String!",
 	"severity: String!",
-	"summary: String!",
-	"suppressActivation: Time",
-	"suppressExpiry: Time",
-	"suppressed: Boolean!",
+	"snoozeExpiry: Time",
+	"snoozeStart: Time",
+	"snoozed: Boolean!",
 	"unusedVarSink(query: String): Int",
 	"vectors: EmbeddedVulnerabilityVectors",
+
+	// deprecated fields
+	"suppressActivation: Time @deprecated(reason: \"use 'snoozeStart'\")",
+	"suppressExpiry: Time @deprecated(reason: \"use 'snoozeExpiry'\")",
+	"suppressed: Boolean! @deprecated(reason: \"use 'snoozed'\")",
+
+	// CVEInfo fields
+	"createdAt: Time @deprecated(reason: \"use 'cveBaseInfo'\")",
+	"cve: String! @deprecated(reason: \"use 'cveBaseInfo'\")",
+	"lastModified: Time @deprecated(reason: \"use 'cveBaseInfo'\")",
+	"link: String! @deprecated(reason: \"use 'cveBaseInfo'\")",
+	"publishedOn: Time @deprecated(reason: \"use 'cveBaseInfo'\")",
+	"scoreVersion: String! @deprecated(reason: \"use 'cveBaseInfo'\")",
+	"summary: String! @deprecated(reason: \"use 'cveBaseInfo'\")",
```

Comment:
I do not recommend deprecating these fields because it allows us to be agnostic of a stored object structure. In short, the service response structure need not be exact as the stored structure.